recipes/patool/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "patool" %}
+{% set version = "1.12" %}
+{% set sha256 = "e3180cf8bfe13bedbcf6f5628452fca0c2c84a3b5ae8c2d3f55720ea04cb1097" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
```

Comment:
It looks like `patools` can be used from a command line interface. Could you add a `commands` test for this:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands